templates/vhost/_rewrite.erb

```diff
@@ -1,6 +1,9 @@
 <%- if @rewrites -%>
   ## Rewrite rules
   RewriteEngine On
+  <%- if @rewriteoptions %>
+  RewriteOptions <%= rewriteoptions %>
```

Comment:
i'd also make this
```ruby
[@rewrite_options].flatten.join(' ')
```
this way we can pass both, a single string, or an array of strings, transparently.